pkg/statistics/handle/syncload/stats_syncload.go

```diff
@@ -254,7 +271,9 @@ func (s *statsSyncLoad) SubLoadWorker(sctx sessionctx.Context, exit chan struct{
 //   - If the task is handled successfully, return nil, nil.
 //   - If the task is timeout, return the task and nil. The caller should retry the timeout task without sleep.
 //   - If the task is failed, return the task, error. The caller should retry the timeout task with sleep.
-func (s *statsSyncLoad) HandleOneTask(sctx sessionctx.Context, lastTask *statstypes.NeededItemTask, exit chan struct{}) (task *statstypes.NeededItemTask, err error) {
+//
+// TODO: remove this session context, it's not necessary.
```

Comment:
I will remove it after https://github.com/pingcap/tidb/pull/58825

Because they both update the context usage here.